pandas/tests/io/generate_legacy_storage_files.py

```diff
@@ -86,6 +84,12 @@
     YearEnd,
 )
 
+try:
```

Comment:
Do you know which version this was changed in? Would be preferable to change import based on pandas version so we know when to clean up